libr/core/cmd_anal.c

```diff
@@ -7449,6 +7459,9 @@ static bool cmd_anal_refs(RCore *core, const char *input) {
 				r_cons_println (pj_string (pj));
 				pj_free (pj);
 			}
```

Comment:
open else in the same line as the }

Reply:
As @karliss had pointed out, in this case, `axt?` doesn't print anything if reference list is not empty, I tried out [this](https://github.com/radareorg/radare2/pull/17389#discussion_r464032768)
Which one should I go with?